src/transactions/transactions.util.ts

```diff
@@ -29,11 +29,10 @@ export type TransactionDetails = {
   status: TransactionStatus;
   fees: Fees;
   supportsSubsidy: boolean;
-  payingAccount: {
-    address: PayingAccount['account']['address'];
-    balance: BigNumber;
-    type: PayingAccount['type'];
-  };
+  payingAccount: Pick<PayingAccount, 'type'> &
+    Pick<Account, 'address'> & {
+      balance: BigNumber;
+    };
```

Comment:
```suggestion
  payingAccount: {
      type: PayingAccountType;
      address: string;
      balance: BigNumber;
    };
```
Why not directly use the types instead of making it dependent? Looks pretty straight forward to me